docs/standards.md

```diff
@@ -197,15 +198,140 @@ Things to take note of:
+
+We repopulate the test server before each test to ensure consistency. [setup.spec.js](../tests/setup.spec.js) contains the code for that. The `storeAll` and `dropAll` functions call test util functions that store and drop specific collections. For example, `account.test.util.js` contains the code a `storeAll` function that inserts all the test account documents into the test database.
+
+#### Util files and Test Database Population
+
+##### Motivation
+
+We wanted to have a scalable way to create new entities that properly reference each other. The biggest challenge lies in creating enough accounts that can be properly referenced during specific tests. 
+
+##### Util.js
+
+Account.util.js contains the test data for accounts. When an account is created, it is placeed within an object whose name references the type of account it will be (ex: Hacker, Sponsor, etc). The object has keys that further define the state of the account. The keys are: `new` for new accounts, `stored` for accounts that already exist, and `invalid` for invalid accounts. Additionally, there are 2 more keys inside `stored` for hacker accounts, which are `team` and `noTeam` for accounts linked to hackers that are on a team, or not on a team respectively.
+
+```javascript
+let hackerAccounts = {
+    new: createNAccounts(10, {
+        "accountType": Constants.HACKER,
+        "confirmed": true,
+    }),
+    stored: {
+        team: createNAccounts(10, {
+            "accountType": Constants.HACKER,
+            "confirmed": true,
+        }),
+        noTeam: createNAccounts(10, {
+            "accountType": Constants.HACKER,
+            "confirmed": true,
+        }),
+    },
+    invalid: createNAccounts(10, {
+        "accountType": Constants.HACKER
+    })
+};
+```
+
+In this example the `new` accounts are accounts that exist, but the hacker objects have not been created. The `stored.team` accounts are those linked to a hacker object that is in a team. The `stored.noTeam` accounts link to  a hacker that is not in a team. The `invalid` accounts are created accounts that are linked to a hacker object that does not fit with the Hacker schema. The invalid accounts are used to test fail cases. The value for each key is an array of account objects. 
+
+On the other end of the account-hacker link, [hacker.util.js](../tests/util/hacker.test.util.js) contains the hacker data in the form of hacker objects. These hacker objects have an `accountId` attribute which references an account's `_id`. The matching between the hacker object and the respective account object it needs to link to is also done by nomenclature. For example, a hacker on a team would be called `TeamHackerX` where X is a number. This hacker's account object would be within the array specified by `hackerAccounts.stored.team`. The specific account object is referenced by its index in the array. That index is the same as the value X in the name of the hacker object.
```

Comment:
Include code for this example, with a value for X. It's hard to follow with just a verbal description.